pkg/controller/authentication/authentication_controller.go

```diff
@@ -256,7 +256,7 @@ func (r *ReconcileAuthentication) Reconcile(ctx context.Context, request reconci
 	// Ensure the allow-tigera tier exists, before rendering any network policies within it.
 	if err := r.client.Get(ctx, client.ObjectKey{Name: networkpolicy.TigeraComponentTierName}, &v3.Tier{}); err != nil {
 		if errors.IsNotFound(err) {
-			r.status.SetDegraded(oprv1.ResourceNotReady, "Waiting for allow-tigera tier to be created", err, reqLogger)
+			r.status.SetDegraded(oprv1.ResourceNotReady, "Waiting for allow-tigera tier to be created, see the TigeraStatus of 'tiers' for more information", err, reqLogger)
```

Comment:
Very nitty, but this reads a bit odd to me - feels like it should be "see the 'tiers' TigeraStatus for more information"

Or maybe even:

> Waiting for allow-tigera tier to be created: see 'kubectl describe tigerastatus tiers" for more information